examples/flatris/browserify/prepare-modules.js

```diff
@@ -0,0 +1,24 @@
+export function prepareComponents(modules) {
+  const components = {};
+  Object.keys(modules).forEach(componentName => {
+    if (componentName.indexOf('_') !== 0) {
+      components[componentName] = () => modules[componentName];
+    }
+  });
+  return components;
+}
+
+export function prepareFixtures(modules, components) {
```

Comment:
Yes! This has been on my mind for a while and has been [raised by many people](https://github.com/skidding/react-cosmos/issues/182). I plan to approach this as well in the future, maybe even included in 1.0, but the current structure will need to continue to work as well.

This is where it can be integrated in the webpack CLI: https://github.com/skidding/react-cosmos/blob/1432aeb487aaae938d767f2de3e0b71309c23170/packages/react-cosmos-webpack/src/build-module-paths.js#L9

E.g.
```js
// Current structure
const fixturesRootPathA = `${componentsRootPath}/__fixtures__/${componentCleanPath}`;
// Nested structure (like you proposed)
const fixturesRootPathB = `${componentsRootPath}/${componentCleanPath}/__fixtures__`;
```

I want to merge this PR before adding that functionality because I want to integrate Cosmos in a repo where it already works. If you want we can work together on adding nested fixture func in a subsequent PR.  👍